cmd/fanal/main.go

```diff
@@ -62,46 +59,70 @@ func run() (err error) {
 				Name:    "image",
 				Aliases: []string{"img"},
 				Usage:   "inspect a container image",
-				Action:  globalOption(ctx, fsCache, imageAction),
+				Action:  globalOption(ctx, imageAction),
 			},
 			{
 				Name:    "archive",
 				Aliases: []string{"ar"},
 				Usage:   "inspect an image archive",
-				Action:  globalOption(ctx, fsCache, archiveAction),
+				Action:  globalOption(ctx, archiveAction),
 			},
 			{
 				Name:    "filesystem",
 				Aliases: []string{"fs"},
 				Usage:   "inspect a local directory",
-				Action:  globalOption(ctx, fsCache, fsAction),
+				Action:  globalOption(ctx, fsAction),
 			},
 			{
 				Name:    "repository",
 				Aliases: []string{"repo"},
 				Usage:   "inspect a remote repository",
-				Action:  globalOption(ctx, fsCache, repoAction),
+				Action:  globalOption(ctx, repoAction),
 			},
 		},
 		Flags: []cli.Flag{
 			&cli.BoolFlag{Name: "clear", Aliases: []string{"s"}},
+			&cli.StringFlag{
+				Name:    "cache",
+				Aliases: []string{"c"},
+				Usage:   "cache backend (e.g. redis://localhost:6379)",
```

Comment:
Currently it's file system path or Redis URL right? It's out of scope, but I was wondering if we could configure S3 backend in similar way?

Reply:
Yes, we can easily support S3 in the same way. But for now, Aqua SaaS is the only user for S3. If an OSS user requests S3 support via CLI options, we'll do that.